### DISCUSSION TITLE
如何屏蔽emoji面板

### DISCUSSION THREAD

**Discussion by 250king** [halo-dev/halo#6552] [Q&A]
是这样子的，鄙站是有备案的，前不久用贵项目来搭建网站，后来在点击评论区emoji面板的时候就发现有一些比较敏感的东西需要屏蔽，避免产生不必要的麻烦，有无家人指点一下这个点的东西是要怎么去掉的？

**Comment by JohnNiang** [comment]
建议去 https://github.com/halo-dev/plugin-comment-widget 提交 Issue 或者 PR 来实现你的需求。
